src/codeql.ts

```diff
@@ -49,31 +51,34 @@ libraryPathDependencies: codeql-${language}`
   );
   fs.writeFileSync(queryFile, query);
 
-  await exec.exec(codeql, [
+  await exec(codeql, [
     "query",
     "run",
     `--database=${database}`,
     `--output=${bqrs}`,
     queryFile,
   ]);
 
-  await exec.exec(codeql, [
```

Comment:
I think these depend on (or cannot run at same time as) the previous exec, but I was at least able to allow these to run in parallel.

Reply:
Ah yes, the results they want to decode won't be there yet. Nice one improving the latter two calls.